src/docs/data/builders/tree.ts

```diff
@@ -0,0 +1,183 @@
+import { DESCRIPTIONS, KBD, PROPS } from '$docs/constants';
+import type { KeyboardSchema } from '$docs/types';
+import { builderSchema, elementSchema } from '$docs/utils';
+import type { BuilderData } from '.';
+
+const BUILDER_NAME = 'tree-view';
+
+const builder = builderSchema(BUILDER_NAME, {
+	title: 'createTreeViewBuilder',
+	description: DESCRIPTIONS.BUILDER('tree-view'),
+	props: [
+		PROPS.FORCE_VISIBLE
+	],
+	elements: [
+		{
+			name: 'tree',
+			description: 'The builder store used to create the tree.',
+			link: '#tree',
+		},
+		{
+			name: 'label',
+			description: 'The builder store used to create the label.',
+			link: '#label',
+		},
+		{
+			name: 'item',
+			description: 'The builder store used to create a tree item.',
+			link: '#item',
+		},
+		{
+			name: 'group',
+			description: 'The builder store used to create a tree group.',
+		}
+	],
+	states: [
+		{
+			name: 'collapsedItems',
+			description: 'The list of IDs of items, who have a group that is not expanded.'
+		}
+	],
+	helpers: [
+		{
+			name: 'focusedItem',
+			description: 'The list item that is currently focused.'
+		},
+		{
+			name: 'selectedItem',
+			description: 'The list item that is currently selected.'
+		}
+	]
```

Comment:
These should be states, a helper is usually a function (so if we had a function `isFocused`, that'd be a helper)